doc/api/cluster.md

```diff
@@ -459,6 +459,21 @@ if (cluster.isMaster) {
 }
 ```
 
+The optional `callback` is a function that is invoked after the message is
+sent but before the worker or master may have received it.
+The function is called with a single argument: `null` on success,
+or an [`Error`][] object on failure.
+
+If no `callback` function is provided and the message cannot be sent, an
+`'error'` event will be emitted by the [`Worker`][] object. This can
+happen, for instance, when the worker has already exited.
+
+
+`worker.send()` will throw an error if the channel has closed or when the
+backlog of unsent messages exceeds a threshold that makes it unwise to send
```

Comment:
This isn't correct. For master, worker.send basically goes through a couple layers, and ends up being a direct call of https://nodejs.org/api/child_process.html#child_process_subprocess_send_message_sendhandle_options_callback. I suggest redirecting to those docs from here.

I'd have to go spelunking through the code, but I strongly suspect that in child processes, .send() is same as process.send(), and that ends up at an internal function that also has the same interface as the ChildProcess.send(). forking a child and doing console.log(process.send) might be easier than reading the code.

That .send  throws when unsent messages exceeds a backlog is definitely not the case, unless I very, very much misread the code.